### ISSUE TITLE
Code Crash in step 2 and 3 while reproducing inference process with the instructions in README.md

### ISSUE THREAD

**Issue by YanzhenHou** [tencent-ailab/Interformer#7]
When we were trying to run  your tool interformer , we encountered a failure during the execution process. We found that the command in the third step failed to run. After examining the source code, we changed the command  `OMP_NUM_THREADS="64,64" python docking/reconstruct_ligands.py -y --cwd $DOCK_FOLDER -y --find_all find` in the third step to `OMP_NUM_THREADS="64,64" python docking/reconstruct_ligands.py -y --cwd $DOCK_FOLDER -y -i 2qbr find`. Then, an error occurred as shown in the diagram. Upon further inspection, we discovered that there was no 'complex' folder within the 'energy_output' directory, which was supposed to be generated by the second step. Additionally, upon reviewing the 'inference.py' file, we noticed that there was no definition for writing to the './complex' folder. The output of the second step, './ligand_reconstruction', also differed from what was defined in the README.md file.

<img width="1437" alt="image" src="https://github.com/user-attachments/assets/1a162f8e-650a-4740-ade1-4a743c190c52" />

**Comment by lkfo415579** [comment]
The second step should generate an energy output folder. Could you provide the log from this step? And we recommend you follow the instruction step by step at the begining.

**Comment by YanzhenHou** [comment]
[cid:f5e201ad-50b7-4fd7-9135-d099a705da83]

it looks like we didn't get the log file.

Thanks a lot for your help, very  timely reply, we are using the tools urgently, and I hope it can be very popularly and widely used.
Yanzhen
________________________________

**Comment by lkfo415579** [comment]
@YanzhenHou Yes, could you follow the instructions do it again? provide some logs here.

**Comment by YanzhenHou** [comment]
I think the log file is printed on screen , as follow:

[cid:6456e11f-dc1a-437d-8e2a-7e8703a18d59]

[cid:0c56eb3e-191f-4e95-a8da-d0ff0d453bb1]


[cid:7e3e1f1e-1c2d-4846-aa28-ea22da2f7b45]
________________________________

**Comment by YanzhenHou** [comment]
I think the log file is printed on screen , as follow:

<img width="895" alt="image" src="https://github.com/user-attachments/assets/551ee850-1fea-42b9-bcdb-022ca088b09d" />

<img width="897" alt="image" src="https://github.com/user-attachments/assets/b52455df-9066-4355-b3dc-06489bca6c0f" />


![image](https://github.com/user-attachments/assets/15fc2956-136c-4b71-a7ff-43f8c1f46b17)

**Comment by YanzhenHou** [comment]
any suggestion?  how to solve the problem ?

**Comment by lkfo415579** [comment]
@YanzhenHou You missed the model checkpoint. Please download it and place it in the project directory.

**Comment by YanzhenHou** [comment]
Hello,

When I run the code, I want to try it with our own examples. So, I would like to ask, the input only requires the protein's pdb and the small molecule's sdf format, right? Regarding the output results, what do you think are the most important? I mean, which output results are helpful for us to assess and filter small molecules?"

Regarding the output results, I've noticed different predictive properties such as rmsd, iner-energy, intra-energy, etc. What do they mean respectively? Or is there a tutorial that provides a detailed interpretation of what the output results represent?

Looking forward to your prompt reply ！
great thanks,



________________________________

**Comment by lkfo415579** [comment]
@YanzhenHou Yes, if you only need to dock one molecule. The properties in the summary csv can be ignored as they are not useful for non-developers. Have you scored the docking poses you just created? The pose score and affinity value can provide additional information to help you select the most favorite molecules. In my opinion, you should do sorting by pose score and then selecting the molecules with high affinity.

**Comment by YanzhenHou** [comment]
OK， great Thanks ! 🌹  Thank you very much for your prompt reply each time.  This is definitely a good impression for the user and one of the best experiences.  Really hope your tools become very popular and loved by more users...

**Comment by YanzhenHou** [comment]
Hello! I'm sorry to bother you again ！when preparing the protein, I encountered the following problem, could you please help me debug?
great thanks !
<img width="1435" alt="image" src="https://github.com/user-attachments/assets/40c9dcef-e9f0-4235-9598-edf3fd9b0de4" />

**Comment by lkfo415579** [comment]
It won't affect the preparation of protein, because I got the same error message as well.

**Comment by YanzhenHou** [comment]
OK ! thanks a lot for your quick reply,  (1) so you mean this 'EEROR' won't affect the following steps ? Even with the ERROR prompts,  actually the preparation of protein is still correct ? right ?
(2) I have another question, compared with autoDock Vina, in vina, there is one step, that we need to set a docking box (the box usually covers the pocket), so here ,   in your tools interformer, when I give the protein, how does the procedure knows where the pocket is ? 
Looking forward for your message soon!
Great thanks !

**Comment by YanzhenHou** [comment]
also, when I continue to run with out own protein and small molecules,  I encountered the following problems:
Looking forward for your reply soon!
Great thanks !

<img width="1437" alt="image" src="https://github.com/user-attachments/assets/a1e3bc3f-5464-4f47-bef7-4e29cab27b43" />

<img width="1439" alt="image" src="https://github.com/user-attachments/assets/0bf35bcb-5100-420b-90d7-b92db9da1b36" />

**Comment by lkfo415579** [comment]
@YanzhenHou (1) Yes. (2) The binding pocket is identified using the reference ligand you input (ligand/$PDB_docked.sdf). The program will then consider the residues near the reference ligand as the binding pocket.

`extract_pocket_by_ligand.py` This script uses a reference ligand to define a binding pocket. However, it appears that the corresponding reference ligand (5000_docked.sdf) can not be found.

**Comment by YanzhenHou** [comment]
Thanks a lot for your quick reply !!
OK, so you mean, when I prepare a protein and ligand (in advance, I should put the ligand approximatly in a certain position of the pocket, for example, I can realize this by other simple docking tools autoDock vina, or use AI prediction tools, like alphaFold3 .etc. )? Or could you please provide better way and tools to put a ligand in the known pocket ?

Anyway, At least, the position (or coordinates) of ligand should be around the pocket I want, right? This might be the reason of failture, because the ligand and protein I use is not in same Coordinate system, they are far away.....

By the way, I didn't mean to offend you, may I have your wechat ? so that when encountering bug, we can have more efficient communication ?  Thanks, 

Looking forward for your reply soon!
Great thanks !

**Comment by YanzhenHou** [comment]
Further, I tried again with another example, the ligand is already docked in the pocket by vina tools, and then I use the docked ligand and protein, as input of interformer, when run  "python tools/extract_pocket_by_ligand.py examples/raw/pocket/ examples/ligand/ 1" , fails again.
<img width="1430" alt="image" src="https://github.com/user-attachments/assets/2c2ec48b-63ae-4f6e-997b-afd1d0f0186e" />

**Comment by lkfo415579** [comment]
@YanzhenHou yes, you can send your WeChat id to my email (mosquitolkfo@gmail.com), then we can have a quick conversation in Chinese.
